includes/class-sensei-course.php

```diff
@@ -1472,7 +1472,10 @@ public function load_user_courses_content( $user = false ) {
 					$lesson_count = 1;
 
 				} // End If Statement
-				$active_html .= '<span class="course-lesson-count">' . esc_html( $lesson_count ) . '&nbsp;' . esc_html__( 'Lessons', 'sensei-lms' ) . '</span>';
+				$active_html .= '<span class="course-lesson-count">' .
+					// translators: Placeholder %d is the lesson count.
+					sprintf( esc_html( _n( '%d Lesson', '%d Lessons', $lesson_count, 'sensei-lms' ) ), $lesson_count ) .
```

Comment:
I wonder if we should flip the `sprintf` with the `esc_html` here as well so that we follow the best practice of escaping as late as possible. In this case it won't matter too much since we're pretty confident that the value of `$lesson_count` that's being swapped in is a number, but it does seem safer to get the formatted string first and then escape after that.

Ditto for all other lines that do this.